pkg/ingester/metrics.go

```diff
@@ -113,6 +114,11 @@ func newIngesterMetrics(r prometheus.Registerer, createMetricsConflictingWithTSD
 			// Could easily return 10m samples per query - 10*(8^(8-1)) = 20.9m.
 			Buckets: prometheus.ExponentialBuckets(10, 8, 8),
 		}),
+		queriedExemplars: promauto.With(r).NewHistogram(prometheus.HistogramOpts{
+			Name:    "cortex_ingester_queried_exemplars",
+			Help:    "The total number of exemplars returned from queries.",
+			Buckets: prometheus.ExponentialBuckets(10, 5, 5),
```

Comment:
**suggestion(optional):** I find comments like this helpful, may be worth adding.

```suggestion
			// A reasonable upper bound is around 6k - 10*(5^(5-1)) = 6250
			Buckets: prometheus.ExponentialBuckets(10, 5, 5),
```